browser/javascript/react-app/src/ampli.test.js

```diff
@@ -0,0 +1,92 @@
+import { Ampli, ApiKey } from './ampli';
+
+describe('Ampli Browser JS SDK tests', () => {
```

Comment:
We are adding unit test for generated ampli? Seems some simple ones.

Reply:
Yep, these will allow us to quick run sanity checks after making updates to codegen. We have the snapshot tests in the SG server but they don't verify the generated code works.